README.md

```diff
@@ -10,7 +10,7 @@ ore-path <String>] [--ice-server <String>...] [--peer <String>...] [--no-trusted
 --graphql-host <String>] [--graphql-port <Nullable`1>] [--libplanet-node] [--workers <Int32>] [--confirmations <Int32>] [--max-transactions <Int32>] [--strict-rendering] [--dev] [--dev.block-interval <Int32>] [--dev.reorg-interval <Int32>] [--log-m
 inimum-level <String>] [--aws-cognito-identity <String>] [--aws-access-key <String>] [--aws-secret-key <String>] [--aws-region <String>] [--help] [--version]
 
-Run standalone application with options.
+Run headless application with options.
```

Comment:
You MUST update the string in [Program.cs#L48](https://github.com/planetarium/NineChronicles.Headless/blob/a86c323caf2c8f2058590e36371f041f40a90737/NineChronicles.Headless.Executable/Program.cs#L48)  if you want update this line.

Reply:
I see, I'll put a PR for that update!